internal/command/term/format.go

```diff
@@ -0,0 +1,18 @@
+package term
+
+import (
+	"fmt"
+	"time"
+)
+
+func BytesToMib(bytes uint64) string {
```

Comment:
Optional: Wouldn't it make sense to replace this function by

```go
func FormatSize(sizeInBytes int64) {
    if sizeInBytes < 1024 {
        return fmt.Sprintf("%d B", sizeInBytes)
    } else if sizeInBytes < 1024 * 1024 {
        return fmt.Sprintf("%.3f kiB", float64(bytes)/1024)
    } else if sizeInBytes < 1024 * 1024 * 1024 {
        return fmt.Sprintf("%.3f MiB", float64(bytes)/1024)
    } else ...
}
```

Maybe receive the `B` part is parameter, so it would also be usable for transfer rates (eg: `MiB/s`).

Reply:
brilliant :-)
thanks!